addons/resource.language.en_gb/resources/strings.po

```diff
@@ -4783,7 +4783,17 @@ msgctxt "#2104"
 msgid "Check the log for more information."
 msgstr ""
 
-#empty strings from id 2105 to 9999
+#empty strings from id 2105 to 2202
+
+msgctxt "#2203"
+msgid "Fall back to DTS for DTS-HD formats"
```

Comment:
its not more intuitive describe this option as "Output DTS core only" instead of fallback...?

Reply:
I was thinking keep it simple, such as:

Use DTS Core

As there's the help description for further info.

Reply:
`Use DTS Core` sounds good, thanks. I'll update the text.